### PR TITLE
[Behat][API] Common saving within contexts

### PR DESCRIPTION
**Pull request by Rafikooo**
| Q               | A                                                            |
|-----------------|--------------------------------------------------------------|
| Branch?         | 1.13                   |
| Bug fix?        | no                                                       |
| New feature?    | yes                                                       |
| BC breaks?      | no                                                      |
| Deprecations?   | no |
| License         | MIT                                                          |

This PR introduces a single method for saving resources (API only), which eliminates the name conflict of saving methods when using multiple contexts within a single suite

### CODE REVIEW

File: src/Sylius/Behat/Context/Api/Common/SaveContext.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Common;
```
Comment (GSadee) [inline src/Sylius/Behat/Context/Api/Common/SaveContext.php:14]: Not sure if this additional namespace is necessary here:
```suggestion
namespace Sylius\Behat\Context\Api;
```

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Rafał! :tada:

File: src/Sylius/Behat/Context/Api/Admin/ManagingAdministratorsContext.php [hunk 2]
```diff
@@ -127,15 +127,6 @@ public function iAddIt(): void
         $this->client->create();
     }
 
-    /**
-     * @When I save my changes
-     */
-    public function iSaveMyChanges(): void
-    {
-        $response = $this->client->update();
-        $this->responseChecker->isUpdateSuccessful($response);
```
Comment (coldic3) [inline src/Sylius/Behat/Context/Api/Admin/ManagingAdministratorsContext.php:136]: This check is missing after refactoring.

Comment (Rafikooo) [reply src/Sylius/Behat/Context/Api/Admin/ManagingAdministratorsContext.php:136]: I removed it on purpose, as `isUpdateSuccessful` doesn't have any assertions, it's simple bool function which result isn't processed further in any way

**Pull request merged**